workspaces/cli-shared/src/diffs/diffs.ts

```diff
@@ -3,42 +3,45 @@
 import { IShapeTrail } from './shape-trail';
 import { IInteractionTrail } from './interaction-trail';
 import { IRequestSpecTrail } from './request-spec-trail';
-//@ts-ignore
-import keymirror from 'keymirror';
 import { IJsonTrail } from './json-trail';
 // Top-level Diffs
 
 export type IDiff =
+  | IUnmatchedQueryParametersShape
   | IUnmatchedResponseBodyShape
   | IUnmatchedRequestBodyShape
   | IUnmatchedRequestUrl
   | IUnmatchedRequestMethod
+  | IUnmatchedQueryParameters
   | IUnmatchedResponseBodyContentType
   | IUnmatchedRequestBodyContentType
   | IUnmatchedResponseStatusCode;
 
-export const DiffTypes: {
-  UnmatchedRequestUrl: string;
-  UnmatchedRequestBodyShape: string;
-  UnmatchedRequestMethod: string;
-  UnmatchedResponseBodyShape: string;
-  UnmatchedResponseBodyContentType: string;
-  UnmatchedResponseStatusCode: string;
-  UnmatchedRequestBodyContentType: string;
-} = keymirror({
-  UnmatchedResponseBodyShape: null,
-  UnmatchedRequestBodyShape: null,
-  UnmatchedRequestUrl: null,
-  UnmatchedRequestMethod: null,
-  UnmatchedResponseBodyContentType: null,
-  UnmatchedRequestBodyContentType: null,
-  UnmatchedResponseStatusCode: null,
-});
+export const DiffTypes = {
+  UnmatchedQueryParametersShape: 'UnmatchedQueryParametersShape',
+  UnmatchedResponseBodyShape: 'UnmatchedResponseBodyShape',
+  UnmatchedRequestBodyShape: 'UnmatchedRequestBodyShape',
+  UnmatchedRequestUrl: 'UnmatchedRequestUrl',
+  UnmatchedRequestMethod: 'UnmatchedRequestMethod',
+  UnmatchedQueryParameters: 'UnmatchedQueryParameters',
+  UnmatchedResponseBodyContentType: 'UnmatchedResponseBodyContentType',
+  UnmatchedRequestBodyContentType: 'UnmatchedRequestBodyContentType',
+  UnmatchedResponseStatusCode: 'UnmatchedResponseStatusCode',
+};
 
 export type IDiffWithShapeDiff =
+  | IUnmatchedQueryParametersShape
   | IUnmatchedResponseBodyShape
   | IUnmatchedRequestBodyShape;
 
+export interface IUnmatchedQueryParametersShape {
+  UnmatchedQueryParametersShape: {
```

Comment:
I think this is missing from your PR in #950 - but we can modify either (this is similar to the request/response body shape diffs